rules/integrations/aws/persistence_route_table_modified_or_deleted.toml

```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2021/06/05"
+maturity = "production"
+updated_date = "2021/10/05"
+integration = "aws"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = "Identifies when an AWS Route Table has been modified or deleted."
+false_positives = ["Automated processes that uses Terraform may lead to false positives."]
+false_positives = [
```

Comment:
@austinsonger duplicated false_positives instance, we need to put this into one.